features.tex

```diff
@@ -17,6 +17,10 @@ \subsection{Matrices}
 % Physics module (some sampling, to show that it is there)
 \subsection{Physics}
 
+% Series module (Formal Power Series, Fourier Series)
+\subsection{Series}
+\input{series.tex}
```

Comment:
This imports content of another .tex file. This way fewer merge conflicts for everyone.